src/math/smek_vec.h

```diff
@@ -269,6 +275,16 @@ T operator-=(T &a, const T &b) {
     return a;
 }
 
+template <VectorType T>
+T operator*=(T &a, const real &b) {
+    a = a * b;
+}
+
+template <VectorType T>
+T operator/=(T &a, const real &b) {
+    a = a / b;
+}
+
```

Comment:
I agree with the compiler here, should be `void`.

Reply:
I think consistency is better. All other operators have a return value.